src/components/AdvancedSearch/AdvancedSearch.tsx

```diff
@@ -162,104 +189,134 @@ const AdvancedSearch: React.FC<{
+  const inputTermRows: { key: string; label: string }[][] = inputTerms.reduce(
+    function (rows, key, index) {
+      return (
+        (index % 2 === 0
+          ? rows.push([key])
+          : rows[rows.length - 1].push(key)) && rows
+      );
+    },
+    []
+  );
+
   return (
-    <>
-      <div className="content-top">
-        <DS.Breadcrumbs
-          modifiers={["space-under"]}
-          breadcrumbs={[{ url: "/", text: breadcrumbTitles.home }]}
+    <Template>
+      <TemplateBreakout>
+        <Breadcrumbs
+          breadcrumbsType={BreadcrumbsTypes.Research}
+          breadcrumbsData={[
+            ...defaultBreadcrumbs,
+            { url: "/advanced-search", text: breadcrumbTitles.advancedSearch },
+          ]}
         />
-      </div>
-      <div className="content-primary advanced-search">
-        <DS.Heading level={1}>Advanced Search</DS.Heading>
-
-        <form
-          onSubmit={(e) => {
-            submit(e);
-          }}
-        >
+      </TemplateBreakout>
+      <TemplateContent>
+        <TemplateContentTop>
+          <Heading level={HeadingLevels.One}>Advanced Search</Heading>
           {emptySearchError && (
-            <DS.HelperErrorText isError={true}>
-              {emptySearchError}
-            </DS.HelperErrorText>
+            <HelperErrorText text={errorMessagesText.emptySearch} isInvalid />
           )}
           {dateRangeError && (
-            <DS.HelperErrorText isError={true}>
-              {dateRangeError}
-            </DS.HelperErrorText>
+            <HelperErrorText text={errorMessagesText.invalidDate} isInvalid />
           )}
-          <fieldset>
-            <div className="search-fields">
-              {inputTerms.map((field: { key: string; label: string }) => {
+        </TemplateContentTop>
+
+        <TemplateContentPrimary>
+          <Form action="/search" method="get" id="search-form">
+            {/* Search Terms */}
+            {inputTermRows.map(
+              (inputTerms: { key: string; label: string }[], i: number) => {
                 return (
-                  <div key={`field-${field.key}`}>
-                    <DS.Label
-                      htmlFor={`${field.key}-input`}
-                      id={`${field.key}-label`}
-                    >
-                      {field.label}
-                    </DS.Label>
-                    <DS.Input
-                      id={`${field.key}-input`}
-                      ariaLabel={`Input for ${field.label}`}
-                      value={
-                        findQueryForField(searchQuery.queries, field.key)
-                          ? findQueryForField(searchQuery.queries, field.key)
-                              .query
-                          : ""
-                      }
-                      onChange={(e) => onQueryChange(e, field.key)}
-                    />
-                  </div>
+                  <FormRow key={`input-row-${i}`}>
```

Comment:
Generally it's better to use an id as a key instead of an iterator. (https://reactjs.org/docs/lists-and-keys.html#keys)   I think here you could use `input-row-${key}`.